### ISSUE TITLE
value in select element not works.

### ISSUE THREAD

**Issue by unbug** [reactjs/express-react-views#37]
According to [why-select-value](https://facebook.github.io/react/docs/forms.html#why-select-value).
```html
<div className="form-group">
                <label htmlFor="role">Role</label>
                <select className="form-control" id="role" name="role" value="Editor">
                  <option value="Register">Register</option>
                  <option value="Editor">Editor</option>
                  <option value="Administrator">Administrator</option>
                </select>
              </div>
```
But the second option not got selected.

**Comment by spicyj** [comment]
This is a React bug that is fixed in master and will be in the 0.14 release.

**Comment by unbug** [comment]
coooool
